app/src/main/java/com/example/translatorapp/screen/translate/TranslateFragment.kt

```diff
@@ -137,20 +140,51 @@ class TranslateFragment :
         }
 
         bind.translate.setOnClickListener {
-            translate(bind.input.text.toString().trim())
+            if (bind.input.text.toString().trim() != "") {
+                translateOnClick(bind.input.text.toString().trim())
+            }
+        }
+
+        bind.copyOutput.setOnClickListener {
+            (activity as MainActivity).copyToClipboard(bind.output.text.toString())
```

Comment:
Không force cast e nhé!
Phải check trc khi cast
```suggestion
            if (activity is MainActivity) activity.copyToClipboard(bind.output.text.toString())
```